cardano-cli/src/Cardano/CLI/EraBased/Options/Governance/Actions.hs

```diff
@@ -277,10 +277,44 @@ dpGovActionProtocolParametersUpdate = \case
       <$> pCommonProtocolParameters
       <*> pAlonzoOnwardsPParams
       <*> pIntroducedInBabbagePParams
+      <*> pIntroducedInConwayPParams
+
+pIntroducedInConwayPParams :: Parser (IntroducedInConwayPParams ledgerera)
+pIntroducedInConwayPParams =
+  IntroducedInConwayPParams
+  <$> pVotingPoolThd
+  <*> pDRepPoolThd
+  <*> pMinCommitteeSize
+  <*> pMinCommitteeTermLength
+  <*> pMinGovActionLifeTime
+  <*> (SJust . toShelleyLovelace <$> pGovActionDeposit)
+  <*> pDRepDeposit
+  <*> ppDRepActivity
+
+ppDRepActivity :: Parser (StrictMaybe EpochNo)
+ppDRepActivity = pure SNothing -- TODO Conway
```

Comment:
TODO Parsers are stubbed to make changes smaller. To be implemented in the next PR.